src/main/java/homework1/task2/ExponentApp.java

```diff
@@ -0,0 +1,20 @@
+package homework1.task2;
+
+public class ExponentApp {
+
+    public static void main(String[] args) {
+        ExponentApp exponent = new ExponentApp();
+        exponent.calculation(2, 40);
+    }
+
+    private long calculation(int number, int exponent) {
```

Comment:
this method doesn't work with negative number (exponent = -3). Change the type of the method long -> double, and add some logic